queries-api/src/main/java/io/corbel/lib/queries/PositionQueryLiteral.java

```diff
@@ -0,0 +1,61 @@
+package io.corbel.lib.queries;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.corbel.lib.queries.exception.MalformedJsonQueryException;
+import io.corbel.lib.queries.exception.QueryMatchingException;
+import io.corbel.lib.queries.model.Coordinates;
+import io.corbel.lib.queries.model.Position;
+import io.corbel.lib.queries.parser.CustomJsonParser;
+import io.corbel.lib.queries.request.QueryLiteral;
+
+/**
+ * @author Alberto J. Rubio
+ *
+ */
+public class PositionQueryLiteral extends QueryLiteral<Position> {
+
+    public PositionQueryLiteral() { super(); }
+
+    public PositionQueryLiteral(CustomJsonParser customJsonParser, JsonNode objectNode)
+            throws MalformedJsonQueryException {
+        super(customJsonParser.readValueAsObject(objectNode, Position.class));
+    }
+
+    public PositionQueryLiteral(Position position) {
+        super(position);
+    }
+
+    @Override
+    protected boolean near(Object object) throws QueryMatchingException {
```

Comment:
It can be useful to compare two objects without asking MongoDB. In addition, other QueryLiterals implement the methods that are available.